Framework/Core/include/Framework/ASoA.h

```diff
@@ -11,6 +11,8 @@
 #ifndef O2_FRAMEWORK_ASOA_H_
 #define O2_FRAMEWORK_ASOA_H_
 
+#include "Framework/Logger.h"
```

Comment:
Please do not include `Framework/Logger.h` in `ASoA.h`

Reply:
Missed this one, thank you.